### PR TITLE
Reindex documents with missing searchable text.

### PR DESCRIPTION
**Pull request by njohner**
Indexing of the SearchableText of documents in solr would sometimes fail because the blob had been updated by the doc properties updater in the meantime. This leads to missing SearchableText. Here we find and reindex such documents.

This upgrade step requires a bump of ftw.solr (https://github.com/4teamwork/ftw.solr/pull/179)

For https://4teamwork.atlassian.net/browse/CA-1361

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)
- Upgrade steps (changes in profile):
  - [x] Make it deferrable if possible
  - [x] Execute as much as possible conditionally

### CODE REVIEW

**Review by lukasgraf — COMMENTED** [review]
Edit: _Sorry, commented on the wrong PR._

File: opengever/core/upgrades/20210325095837_reindex_missing_searchable_text/upgrade.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            fl=["UID", "SearchableText"],
+            rows=1000000)
+        for doc in resp.docs:
+            if "SearchableText" in doc:
+                continue
+
+            obj = unrestrictedUuidToObject(doc['UID'])
+            if not obj:
+                continue
+            print "reindexing", obj
```
Comment (lukasgraf) [inline opengever/core/upgrades/20210325095837_reindex_missing_searchable_text/upgrade.py:32]: This should be a log message instead of `print`.

**Review by lukasgraf — APPROVED** [review]
LGTM 👍 , just a stray `print`

File: opengever/core/upgrades/20210325095837_reindex_missing_searchable_text/upgrade.py (hunk 1 above)
Comment (njohner) [reply opengever/core/upgrades/20210325095837_reindex_missing_searchable_text/upgrade.py]: Right thanks. I added some correct logging instead

**Pull request merged**